docs/section5/Rule5-43_nonprimary.md

```diff
@@ -0,0 +1,45 @@
+# Envelope - Rule 5-43  
+**Schema Version** 0.0.23  
+**Primary Rule:** False
+**Rule ID:** 5-43  
+**Rule Description:** The  proposed roof surfaces shall be modeled using the same solar reflectance as in the user model if the aged test data are available, or equal to 0.7 default reflectance.
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-1(a) Building Envelope Modeling Requirements for the Proposed design  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+  1. The proposed building has a roof surface
+
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. get_opaque_surface_type()
+  2. match_data_element()
+
+## Rule Logic:
+
+- For each building segment in the Proposed model: ```for building_segment_p in P_RMR.building.building_segments:```
+
+  - For each zone in building segment: ```for zone_p in building_segment_p.zones:```
+    
+    - For each surface in zone: ```for surface_p in zone_p.surfaces:```
+
+      - Check if surface is roof, set rule applicability check to True: ```if get_opaque_surface_type(surface_p.id) == "ROOF": rule_applicability_check = TRUE```
+  
+        - Get matching surface in U_RMR: ```surface_u = match_data_element(U_RMR, Surfaces, surface_p.id)```
+        **Rule Assertion:**
+
+        - Case 1: If roof surface solar reflectance in P_RMR matches U_RMR and is equal to 0.3, outcome is PASS: ```if ( surface_p.surface_optical_properties.absorptance_solar_exterior == surface_u.surface_optical_properties.absorptance_solar_exterior ) AND ( surface_p.surface_optical_properties.absorptance_solar_exterior == 0.7 ):
```

Comment:
would be helpful to have a clarifying note here than solar reflectance = 1-solar absorptance.